### DISCUSSION TITLE
Buy premium block

### DISCUSSION THREAD

**Discussion by IRediTOTO** [tremorlabs/tremor-npm#229] [Q&A]
I know you will have premium block. Is there an open sale for new start?

**Comment by severinlandolt** [comment]
Hey @IRediTOTO ! We do not want to rule out that there might be some paid components in the future. For now, we do not have any specific plans or roadmap for paid/premium blocks.

**Comment by join-hq** [comment]
收到了你的信件，我现在没在上网。
